pkg/api/converterfromagentpoolonlyapi.go

```diff
@@ -130,12 +130,37 @@ func convertResourcePurchasePlanToV20180331AgentPoolOnly(api *ResourcePurchasePl
 	v20180331.Publisher = api.Publisher
 }
 
+func convertKubernetesConfigToEnableRBACV20180331AgentPoolOnly(kc *KubernetesConfig) bool {
+	if kc == nil {
+		return false
+	}
+	// Only if both KubernetesConfig.EnableRbac and KubernetesConfig.EnableSecureKubelet true
+	// We think EnableRBAC is on
+	enableRBAC := true
```

Comment:
the code seems pretty complicated to me. why not base on the comments:

enableRBAC := (KubernetesConfig.EnableRbac == true) && (KubernetesConfig.EnableSecureKubelet == true)

Reply:
EnableRbac is *bool !

Reply:
I see, could this work then:
enableRBAC := (KubernetesConfig.EnableRbac != nil) && (*(KubernetesConfig.EnableRbac) == true) && (KubernetesConfig.EnableSecureKubelet != nil) && (*(KubernetesConfig.EnableSecureKubelet) == true)

no big deal, just want to make code a bit more easy to read. :)

Reply:
Made it easier to read.